app/models/fe/concerns/answer_sheet_concern.rb

```diff
@@ -69,8 +69,8 @@ def reference?
       false
     end
 
-    def percent_complete
-      num_questions = question_sheets.inject(0.0) { |sum, qs| qs.nil? ? sum : qs.questions_count + sum }
+    def percent_complete(required_only = false)
+      num_questions = question_sheets.inject(0.0) { |sum, qs| qs.nil? ? sum : qs.questions_count(required_only) + sum }
```

Comment:
Just looking at this now, this doesn't look right -- https://github.com/CruGlobal/fe/pull/61/files#diff-757ede0ee35aad6d0cc88b4605ab03ebR33 has the first param as an answer sheet.  I'll add some tests for percent_complete